examples/md-flexible/tests/ThermostatTest.cpp

```diff
@@ -45,19 +46,54 @@ void ThermostatTest::testBrownianMotion(const Molecule &dummyMolecule, const dou
               0.02);
 }
 
-TEST_F(ThermostatTest, BrownianMotionTest_InitV0) {
-  Molecule dummyMolecule;
+/**
+ * Tests the application of Brownian Motion for the case that the molecule being applied to has no initial velocity.
+ */
+TEST_F(ThermostatTest, BrownianMotionTest_ZeroInitialVel) {
+  ParticleType dummyMolecule;
   testBrownianMotion(dummyMolecule, 1.5);
 }
 
-TEST_F(ThermostatTest, BrownianMotionTest_InitV1) {
-  Molecule dummyMolecule;
+/**
+ * Tests the application of Brownian Motion for the case that the molecule being applied to has non-zero initial
+ * velocity.
+ */
+TEST_F(ThermostatTest, BrownianMotionTest_NonZeroInitialVel) {
+  ParticleType dummyMolecule;
   dummyMolecule.setV({1, 1, 1});
   testBrownianMotion(dummyMolecule, 1.5);
 }
 
+/**
+ * For multi-site molecules, tests the application of Brownian Motion for the case that the molecule being applied to
+ * has non-zero initial angular velocity but zero translational velocity.
+ */
+TEST_F(ThermostatTest, BrownianMotionTest_NonZeroInitialAngVelWithZeroTransVel) {
+#if MD_FLEXIBLE_MODE == MULTISITE
+  ParticleType dummyMolecule;
+  dummyMolecule.setAngularVel({1, 1, 1});
+  testBrownianMotion(dummyMolecule, 1.5);
+#endif
+}
+
+/**
+ * For multi-site molecules, tests the application of Brownian Motion for the case that the molecule being applied to
+ * has non-zero initial angular velocity and non-zero translational velocity.
+ */
+TEST_F(ThermostatTest, BrownianMotionTest_NonZeroInitialAngVelWithNonZeroTransVel) {
+#if MD_FLEXIBLE_MODE == MULTISITE
+  ParticleType dummyMolecule;
+  dummyMolecule.setV({1, 1, 1});
+  dummyMolecule.setAngularVel({1, 1, 1});
+  testBrownianMotion(dummyMolecule, 1.5);
+#endif
+}
+
```

Comment:
This was mostly because I wanted the same tests to exist for single-site, but for them to be bypassed. This isn't something I feel strongly about, so if you would prefer it your way, I'm happy to do so. Alternatively, I could use `GTEST_SKIP` (or whatever the syntax is) to make it clear that these tests are actively skipped.

Reply:
ah no, that's fine.